contracts/handler/Aave/AaveAssetHandler.sol

```diff
@@ -278,6 +332,82 @@ contract AaveAssetHandler is IAssetHandler {
     return (tokenBalance * unusedCollateralPercentage) / 10 ** 18; // Calculate and return the investible balance
   }
 
+  function loanProcessingDex(
+    address vault,
+    address executor,
+    address controller,
+    address receiver,
+    address[] memory lendTokens,
+    uint256 totalCollateral,
+    uint fee,
+    FunctionParameters.FlashLoanData memory flashData
+  ) external view returns (MultiTransaction[] memory transactions, uint256) {
+    // Process swaps and transfers during the loan
+    (
+      MultiTransaction[] memory swapTransactions,
+      uint256 totalFlashAmount
+    ) = swapAndTransferTransactionsUsingDex(vault, executor, flashData);
+
+    // Handle repayment transactions
+    MultiTransaction[] memory repayLoanTransaction = repayTransactions(
+      executor,
+      vault,
+      flashData
+    );
+
+    // Handle withdrawal transactions
+    MultiTransaction[]
+      memory withdrawTransaction = withdrawTransactionsUsingDex(
+        executor,
+        vault,
+        controller,
+        receiver,
+        lendTokens,
+        totalCollateral,
+        fee,
+        flashData
+      );
+
+    // Combine all transactions into one array
```

Comment:
After here it's the same code as in `loanProcessing`

We can put the code into an internal function to avoid duplicates (it will be easier to maintain the code)